servant-client-core/servant-client-core.cabal

```diff
@@ -102,7 +102,7 @@ library
   --
   -- note: mtl lower bound is so low because of GHC-7.8
   build-depends:
-    , base              >=4.14     && <4.21
+      base              >= 4.16.4.0 && < 4.21
     , bytestring        >=0.10.8.1 && <0.13
```

Comment:
Now that GHC 9.2 is the minimum version, it would be nice to revise all bytestring bounds to be `>=0.11`.